### PR TITLE
create OLS folder and add Ethics project folder

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Related to Ethical research guide

* Creates an OLS folder for the teams to store their notes and files

### List of changes proposed in this PR (pull-request)

* creates a dedicated place for OLS projects
* links all the projects that are under OLS-2 for Tuning and shares the GitHub space with the OLS project teams

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@Ismael-KG and @LauraCarter I am resending you both an invitation to join the repo as contributors. you will get a notification. See the bell symbol on the top right. 
image

File: open-life-science-mentoring/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## OLS-2 for Turing projects
+
+Congratulations to the following 6 projects that will join the second cohort (OLS-2) under the collaboration *OLS-2 for Turing*.
+See [this post for the announcement](https://openlifesci.org/posts/2020/09/01/ols2-announcement/) along with the [details of 33 projects](https://openlifesci.org/ols-2/projects-participants/) that will be trained and mentored in OLS-2.
+
+- Ethical Research Framework - _The Turing Way_, Team: @BrainonSilicon, @Ismael-KG, @LauraCarter, Mentors: @jezcope, Anjali Mazumdar , [project link](./ethical-research-framework)
+- sktime - a unified toolbox for machine learning with time series, Team members: @mloning, Mentor: @martinagvilas, [project link](https://github.com/alan-turing-institute/sktime)
+- Turing Data Stories: [project link](https://github.com/alan-turing-institute/TuringDataStories), Team members: @crangelsmith, @DavidBeavan, @samvanstroud, @kevinxufs Mentor: @yochannah, [project link](https://github.com/alan-turing-institute/AutisticaCitizenScience)
+- Embedding Accessibility in The Turing Way Open Source Community Guidance, Team members: @pgadige, Mentor: TBA, [project link](./turing-way-accessibility)
```
Comment (malvikasharan) [inline open-life-science-mentoring/README.md:20]:
```suggestion
- Embedding Accessibility in The Turing Way Open Source Community Guidance, Team members: @pgadige, Mentor: TBA, [project link](./turing-way-accessibility) - TBA
```

**Comment by BrainonSilicon** [comment]
Looks great - will switch from OLS branch to this folder

**Review by BrainonSilicon — APPROVED** [review]
Looks amazing Malvika!

**Pull request merged**